spec/Observable-spec.ts

```diff
@@ -810,6 +810,24 @@ describe('Observable', () => {
         }).not.to.throw();
       });
 
+      it('should call teardown if sync unsubscribed', () => {
```

Comment:
This is the only test that was failing. I just added the other one to cover the other scenario.